Docs: Add hide_edition config to the docs (#73576)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/enterprise-configuration/index.md

Set to complete URL to override Apple/iOS icon.

### hide_edition

> Hides the Grafana edition from being used in the footer if set to "true"

Reviewer: ```suggestion
Set to `true` to remove the Grafana edition from the footer.
```
